### ISSUE TITLE
Brave crashes when opening facebook from the address bar

### ISSUE THREAD

**Issue by pitsi** [brave/brave-browser#29878]
## Description 
As mentioned in the title, after today's upgrade to 1.50.121, brave crashes as soon as I type "fa" in the address bar. If it helps, messenger.com is also sluggish after the upgrade. I have not noticed any similar behavior on other sites yet.

## Steps to Reproduce

   1. Launch brave (preferably under linux) and with some facebook page already visited in the past.
   2. Type "facebook" in the address bar.
   3. Watch brave crash.

## Actual result:
Brave crashes. Instead of screenshot, here is what comes out in the terminal
```
$ brave-browser
[9902:9902:0420/071717.085386:ERROR:chrome_browser_cloud_management_controller.cc(162)] Cloud management controller initialization aborted as CBCM is not enabled.
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[9936:9936:0420/071717.193248:ERROR:gpu_memory_buffer_support_x11.cc(49)] dri3 extension not supported.
[9902:10026:0420/071717.749558:ERROR:object_proxy.cc(623)] Failed to call method: org.freedesktop.DBus.Properties.Get: object_path= /org/freedesktop/UPower: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.UPower was not provided by any .service files
[9902:10026:0420/071717.752566:ERROR:object_proxy.cc(623)] Failed to call method: org.freedesktop.UPower.GetDisplayDevice: object_path= /org/freedesktop/UPower: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.UPower was not provided by any .service files
[9902:10026:0420/071717.759908:ERROR:object_proxy.cc(623)] Failed to call method: org.freedesktop.UPower.EnumerateDevices: object_path= /org/freedesktop/UPower: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.UPower was not provided by any .service files
[9902:9932:0420/071717.949069:ERROR:nss_util.cc(357)] After loading Root Certs, loaded==false: NSS error code: -8018
[9902:9902:0420/071718.374265:ERROR:interface_endpoint_client.cc(695)] Message 0 rejected by interface blink.mojom.WidgetHost
[0420/071722.479441:ERROR:elf_dynamic_array_reader.h(64)] tag not found
/usr/bin/brave-browser: line 48:  9902 Illegal instruction     (core dumped) "$HERE/brave" "$@"
```

## Expected result:
Brave should open facebook.

## Reproduces how often: 
Easily, at least on my end.

## Brave version (brave://version info)
```
Brave	1.50.121 Chromium: 112.0.5615.138 (Official Build) (64-bit) ￼
Revision	2258f2fcaeb07bc95c992b7ca47f0b43ead9865c-refs/branch-heads/5615@{#1282}
OS	Linux
```

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? No idea
- Can you reproduce this issue with the nightly channel? No idea

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? No (tested on chromium 112.0.5615.121)

## Miscellaneous Information:
I launch brave with this set of command line parameters, but nothing changes if I launch it without them.
`brave-browser --disk-cache-dir=/tmp/cache --process-per-site`
Opening facebook from the history (ctrl + h > search for any facebook entry > click) does open the page with no issues.
Same behavior happens on private windows too.

---edit
It just happened on reddit, when I tried opening this reddit thread 
`https://www.reddit.com/r/apolloapp/comments/12ram0f/had_a_few_calls_with_reddit_today_about_the/`
from a comment posted on this one 
`https://old.reddit.com/r/Android/comments/12rmi75/big_change_for_third_party_reddit_apps_as_reddit/`
It gives the same error on the terminal.

---re-edit
The facebook issue does not happen on a guest profile, but reddit one still does.

**Comment by CodeDead** [comment]
All the related issues:
https://github.com/brave/brave-browser/issues/29878
https://github.com/brave/brave-browser/issues/29871

**Comment by pitsi** [comment]
A few minutes ago the issue happened with github as well, but when trying to go to a specific github repo from the address bar.
Instead of hunting down the urls or scenarios under which it happens, I decided to downgrade brave to 1.50.119 which works flawlessly.
Once chromium reaches that version, probably tomorrow or during the weekend, I will check what happens there and report back.

@CodeDead
Thank you for the clarification.

**Comment by pitsi** [comment]
I got the upgrade to chromium 112.0.5615.138 a few minutes ago and I retested it. It has no issues opening facebook or github or reddit. However, because chromium is my third browser (primary = brave, secondary = firefox, third = chromium), I am not logged in on any of those sites and the only addon I use on it is ubo lite.

**Comment by kjozwiak** [comment]
Firstly, I would like to apologies to anyone who's experiencing the above issues and appreciate all the reports and information. It looks like `112.0.5615.138` is causing issues on `Linux` and we suspect it might be due to https://bugs.chromium.org/p/chromium/issues/detail?id=1434194. What usually happens is when we get a chromium bump as per https://chromereleases.googleblog.com/2023/04/stable-channel-update-for-desktop_18.html, we update all our branches and then push the release on the targeted platforms. So for `112.0.5615.138`, both `macOS` & `Windows` were targeted. The update mentions that a `Linux` update will be coming soon. However, when we usually get bumps with zero-day vulnerabilities, we try to update all of our platforms ASAP. So we ended up pushing `112.0.5615.138` on `Linux` as well to help protect our users. However, in hindsight, we probably should have waited.

We're preparing a new update that will basically bump the version from `1.50.121` and revert users to `1.50.119` which was using `112.0.5615.121`. We've received a bunch of confirmations spanning various threads on both reddit and GitHub that reverting to `112.0.5615.121` fixes the issues. We've also took a look at the difference between `112.0.5615.138` & `112.0.5615.121` and see nothing that would cause any issues when reverting. We should have an update available within ~2-3hrs. In the meantime, I would advice `Linux` users not to update to `112.0.5615.138` and wait for the next chromium bump.

We'll also update the various Brave communication channels including reddit and http://community.brave.com to let users know what happened and let them know that an update is on the way that should restore functionality back to normal.
